extensions/github1s/src/adapters/ossinsight/interfaces.ts

```diff
@@ -12,104 +12,82 @@ const resolveDataFromResponse = (response: Response) => {
 	if (response.status < 200 || response.status >= 300) {
 		return [];
 	}
-	return response.json().then((body) => body?.data || []);
+	return response.json().then((body) => body?.data?.rows || []);
 };
 
 export type RepoItem = {
 	collection_names?: string;
 	contributor_logins?: string;
 	description?: string;
-	forks?: number;
+	forks: string;
```

Comment:
Any reason to remove it from being an optional entity?

```suggestion
	forks?: string;
```

Reply:
It is marked as required in the api document of ossinsight